assets/i18n/en_US.json

```diff
@@ -163,13 +163,15 @@
     },
     "installerView": {
         "widgetTitle": "Installer",
-        "installType": "Select install type",
-        "installTypeDescription": "Select the installation type to proceed with.",
+        "installType": "Choose installation method",
+        "installTypeDescription": "To install, you can either mount or install normally. Mounting will install the patched app on top of an existing installation.",
```

Comment:
What's the motivation behind the update of the description here? I believe the existing description is better fitting, the install methods can speak for themselves already

Reply:
Perhaps the "overline" line in ListItems should be used in that case, that would allow all installation methods to have their own miniature description without relying on the dialog's description itself to describe it appropiately both in meaning and sentence structuring

Reply:
![image](https://github.com/ReVanced/revanced-manager/assets/79272171/d3748674-ebb5-4a11-bc32-2cd234248d7e)

The design in Figma can be used, for install methods that benefit from an explanation would use a trailing help icon button

Reply:
This can be allocated into a separate PR as it should not delay the current one rmore.